src/test/groovy/io/codearte/jfairy/producer/person/PersonSpec.groovy

```diff
@@ -214,5 +231,118 @@ class PersonSpec extends Specification {
 			person.passportNumber()
 	}
 
+	def "withFirstName should create person with specific first name"() {
+		when:
+			def person = fairy.person(withFirstName("Specificfirstname"))
```

Comment:
I would like to suggest to fix the `def` issue in another PR after PR #68 is merged or closed. This is to avoid merge conflicts. I can do the fix asap as soon as a decision is made.